go.mod

```diff
@@ -78,11 +79,13 @@ require (
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.4 // indirect
+	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/prometheus/client_golang v1.16.0 // indirect
 	github.com/prometheus/client_model v0.4.0 // indirect
 	github.com/prometheus/common v0.42.0 // indirect
 	github.com/prometheus/procfs v0.10.1 // indirect
+	github.com/r3labs/sse/v2 v2.10.0 // indirect
```

Comment:
@mstoykov I remember you asked about it, can you point me there, please?

Reply:
I guess you mean https://github.com/grafana/xk6-dashboard/issues/108#issuecomment-1855397461 ?

Reply:
@szkiba It would be good to have a quick sketch about an overview of the extension's architecture and add it to the `docs` folder in the `xk6-dashboard`. So we can quickly realize what are the connections in place and the flow of data.

Reply:
@codebien  FYI https://github.com/grafana/xk6-dashboard#how-it-works
Is it what you mean? Or something more detailed?

Reply:
Yes, it's good enough thanks! :tada: Maybe, just as a suggestion it would be good to split the Dashboard readme and have in the main page just a short version with the index as we do here in the k6 repository.

Reply:
Yes, you areright, the readme a bit long now. I'm going to reorganize it, but it will be independent from the release (I'm not planning a release after it)

Reply:
>but it will be independent from the release
Yes, definitely it shouldn't be a blocker for this PR.

Reply:
>  Maybe, just as a suggestion it would be good to split the Dashboard readme

I made the readme a bit more readable...